components/popover/popover.jsx

```diff
@@ -478,7 +475,6 @@ const Popover = createReactClass({
 					aria-describedby={`${this.getId()}-dialog-body`}
 					className={classNames(
 						'slds-popover',
```

Comment:
This is incredibly annoying since it can't be moved up to the dialog level or at least I'm not sure how to go about it since it's difficult to follow all the logic of trapping the focus in the popover and following this up and down through the components, life cycle methods, and refs.

Reply:
I think it's easier to follow from a comparing the HTML nodes here to SLDS if you pass in the whole popover.

Reply:
For now leaving it as is, but after thinking about this some more, I think that I disagree, the Dialog is the Popover. You should be able to render an empty popover. Why should it be `div div.popover #contents` when it can just be `div.popover #contents`. It's a bit annoying because we're trying to unify all these components under the concept of a "Dialog" but SLDS limits us in this fashion. Leaving it as is, because I'm already rushed for time as is.